*: add TiDB basic features and update overview (#2999) (#3098)
pingcap/docs: overview.md

---
title: TiDB Introduction

> summary: Learn about the key features and usage scenarios of TiDB.

Reviewer: ```suggestion
summary: Learn about the key features and use cases of TiDB.
```

category: introduction

> aliases: ['/docs/stable/overview/','/docs/v4.0/overview/','/docs/dev/key-features/','/tidb/dev/key-features']

Reviewer: ```suggestion
aliases: ['/docs/stable/overview/','/docs/v4.0/overview/','/docs/v4.0/key-features/','/tidb/v4.0/key-features']
```

---
title: TiDB Introduction
summary: Learn about the key features and usage scenarios of TiDB.
category: introduction

> <<<<<<< HEAD
> aliases: ['/docs/stable/overview/','/docs/v4.0/overview/']
> =======
> aliases: ['/docs/dev/key-features/','/tidb/dev/key-features','/docs/dev/overview/']
> >>>>>>> 724a55b... *: add TiDB basic features and update overview (#2999)

Reviewer: ```suggestion
aliases: ['/docs/stable/overview/','/docs/v4.0/overview/','/docs/dev/key-features/','/tidb/dev/key-features']
```
